src/w3d/renderer/mapper.cpp

```diff
@@ -882,7 +882,7 @@ BumpEnvTextureMapperClass::BumpEnvTextureMapperClass(const BumpEnvTextureMapperC
 
 inline unsigned long F2DW(float f)
```

Comment:
There are two of these in the codebase?? Can you create an issue to track merging them (and changing it to a dword instead of usigned long)

Reply:
There are actually 3 of these.